src/test/regress/expected/aqumv.out

```diff
@@ -985,6 +985,396 @@ select count(c2), count(*) from aqumv_t2 where c1 > 90;
+  92 |     |     1
+  93 |     |     1
+  99 |     |     1
+  94 |     |     1
+ 100 |     |     1
+  96 |     |     1
+  97 |     |     1
+  98 |     |     1
+  95 |     |     1
+  91 |     |     1
+     |  93 |     1
+     | 101 |     1
+     |  99 |     1
+     |  94 |     1
+     | 100 |     1
+     | 102 |     1
+     |  97 |     1
+     |  96 |     1
+     |  98 |     1
+     |  95 |     1
+(20 rows)
+
+-- ROLLUP
+set local enable_answer_query_using_materialized_views = off;
+explain(costs off, verbose)
+select c1, c3, count(c2) from aqumv_t3 where c1 > 90 group by rollup(c1, c3);
+                                    QUERY PLAN                                     
+-----------------------------------------------------------------------------------
+ GroupAggregate
+   Output: c1, c3, count(c2)
+   Group Key: aqumv_t3.c1, aqumv_t3.c3
+   Group Key: aqumv_t3.c1
+   Group Key: ()
+   ->  Gather Motion 3:1  (slice1; segments: 3)
+         Output: c1, c3, c2
+         Merge Key: c1, c3
+         ->  Sort
+               Output: c1, c3, c2
+               Sort Key: aqumv_t3.c1, aqumv_t3.c3
+               ->  Seq Scan on public.aqumv_t3
+                     Output: c1, c3, c2
+                     Filter: (aqumv_t3.c1 > 90)
+ Settings: enable_answer_query_using_materialized_views = 'off', optimizer = 'off'
+ Optimizer: Postgres query optimizer
+(16 rows)
+
+select c1, c3, count(c2) from aqumv_t3 where c1 > 90 group by rollup(c1, c3);
+ c1  | c3  | count 
+-----+-----+-------
+  91 |  93 |     1
+  91 |  95 |     0
+  91 |     |     1
+  92 |  94 |     1
+  92 |     |     1
+  93 |  95 |     1
+  93 |     |     1
+  94 |  96 |     1
+  94 |     |     1
+  95 |  97 |     1
+  95 |     |     1
+  96 |  98 |     1
+  96 |     |     1
+  97 |  99 |     1
+  97 |     |     1
+  98 | 100 |     1
+  98 |     |     1
+  99 | 101 |     1
+  99 |     |     1
+ 100 | 102 |     1
+ 100 |     |     1
+     |     |    10
+(22 rows)
+
+set local enable_answer_query_using_materialized_views = on;
+explain(costs off, verbose)
+select c1, c3, count(c2) from aqumv_t3 where c1 > 90 group by rollup(c1, c3);
+                                    QUERY PLAN                                    
+----------------------------------------------------------------------------------
+ GroupAggregate
+   Output: mc1, mc3, count(mc2)
+   Group Key: aqumv_mvt3_0.mc1, aqumv_mvt3_0.mc3
+   Group Key: aqumv_mvt3_0.mc1
+   Group Key: ()
+   ->  Gather Motion 3:1  (slice1; segments: 3)
+         Output: mc1, mc3, mc2
+         Merge Key: mc1, mc3
+         ->  Sort
+               Output: mc1, mc3, mc2
+               Sort Key: aqumv_mvt3_0.mc1, aqumv_mvt3_0.mc3
+               ->  Seq Scan on public.aqumv_mvt3_0
+                     Output: mc1, mc3, mc2
+ Settings: enable_answer_query_using_materialized_views = 'on', optimizer = 'off'
+ Optimizer: Postgres query optimizer
+(15 rows)
+
+select c1, c3, count(c2) from aqumv_t3 where c1 > 90 group by rollup(c1, c3);
```

Comment:
without order by , are the result output stable ?

Reply:
Hi, thanks for review.
> without order by, are the result output stable?

No, in general, and different order won't cause diff in our regression test.
But for this case,  the result is stable because the way core codes process rollup.

Reply:
what about `select c1, c3, count(c2) from aqumv_t3 where c1 > 90 group by c1, c3;` QEs may return randomly.

Reply:
Yeah, the result is unordered,it's not rollup.

Reply:
Group by has no semantic of order, what's your concern?